incubator/azuremonitor-containers/templates/omsagent-rs-configmap.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if and (ne .Values.omsagent.secret.key "<your_workspace_key>") (ne .Values.omsagent.secret.wsid "<your_workspace_id>") (ne .Values.omsagent.env.clusterName "<your_cluster_name>")}}
+kind: ConfigMap
+apiVersion: v1
+data:
+  kube.conf: "# Fluentd config file for OMS Docker - cluster components (kubeAPI)\r\n\r\n#Kubernetes
```

Comment:
Is that anyway this can be more readable?

Reply:
Fixed it. Thx for pointing out.